test/integration/MatchingServiceConnectorSpec.scala

```diff
@@ -257,7 +257,7 @@ class MatchingServiceConnectorSpec extends IntegrationTestsSpec with ScalaFuture
       )
       await(matchingServiceConnector.lookup(serviceRequestJson.as[MatchingRequestHolder]))
 
-      eventually(AuditService.verifyXAuditWrite(2))
+      eventually(AuditService.verifyXAuditWrite(1))
```

Comment:
Why are these 1 less than before?

Reply:
Because we had one event for request and one for response. Now those places are handled by one event

Reply:
Makes sense.  Thanks